rllib/core/rl_module/torch/tests/test_torch_rl_module.py

```diff
@@ -154,7 +154,7 @@ def test_forward_train(self):
                     batch.append(
                         {
                             "obs": obs,
-                            "action": action[None] if action.ndim == 0 else action,
+                            "actions": action[None] if action.ndim == 0 else action,
```

Comment:
Shouldn't we use `SampleBatch.OBS/ACTIONS/REWARDS/DONES` here?

Reply:
This wasn't originally written for ingesting sample batches. 

I modified it for that reason, but I don't want to make any more changes in this pr

Reply:
Ok, got it. If it has nothing to do with SampleBatch, then it's fair to leave it.